keyboards/packrat/keymaps/3Uc/keymap.c

```diff
@@ -0,0 +1,103 @@
+/* Copyright 2021 drhigsby
```

Comment:
Invalid keymap name, folder has to be lowercase. eg `3uc`

Reply:
Oh, good to know. I will amend this.